src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskTypesControllerTest.java

```diff
@@ -221,7 +223,8 @@ void should_return_503_with_application_problem_response_when_camunda_api_is_not
         when(camundaServiceApi.getTaskTypesDmnTable(
             anyString(),
             anyString(),
-            anyString()
+            anyString(),
+            anyBoolean()
```

Comment:
One of the AC's is to handle the latest version of the dmn I.e. "about evaluating the latest version of each DMN and there is a flag called `latestVersion`". Can we add a test for this to show we always take the latest?

Reply:
As discussed, the latest version parameter should return the latest version. this managed within Camunda